cg/clients/arnold/api.py

```diff
@@ -1,6 +1,33 @@
 """Module for the arnoldAPIClient."""
 
+import logging
+from http import HTTPStatus
+
+import requests
+from requests import Response
+
+from cg.clients.arnold.dto.create_case_request import CreateCaseRequest
+from cg.clients.arnold.exceptions import ArnoldClientError, ArnoldServerError
+
+LOG = logging.getLogger(__name__)
+
 
 class ArnoldAPIClient:
     def __init__(self, config: dict):
         self.api_url: str = config["api_url"]
+
+    def create_case(self, case: CreateCaseRequest) -> Response:
+        endpoint: str = f"{self.api_url}/case/"
+        post_request_data: CreateCaseRequest = case
+        response: Response = requests.post(endpoint, data=post_request_data)
+        if response.status_code == HTTPStatus.OK:
+            LOG.info(f"Successfully created case for {case.case_id} in Arnold.")
+            return response
+        self._handle_errors(response)
+
+    @staticmethod
+    def _handle_errors(response: Response):
+        if 400 <= response.status_code < 500:
+            raise ArnoldClientError(f"Client error: {response.status_code}")
+        elif 500 <= response.status_code:
+            raise ArnoldServerError(f"Server error: {response.status_code}")
```

Comment:
Are the response codes sufficient or is there value in including the error message? I often prefer having the error message as well, but I don't know how the messages look here.

Reply:
Also might be bad praxis and one should need to look in the Arnold logs to see what went wrong.

Reply:
Fixed it, Arnold already sends a detailed response when an error occurs.  This should be displayed with `request.content`